### DISCUSSION TITLE
Accelerometer replacement? TS100

### DISCUSSION THREAD

**Discussion by mr-bridger** [Ralim/IronOS#1475] [General]
I've got a TS100 that doesn't respond to movement, (Accelerometer not detected warning) it doesnt work on IronOS or on stock firmware, ive not tried a reflow yet, (to be honest, under the scope it looks ok) but wondered, if that isn't successful, what can I replace the accelerometer with?

I'm sure I saw a list of IronOS compatible accelerometers, but cant find it now (maybe I dreamed it lol ) 

I think I found the OG TS100's used a MMA8652FC, but these are not easy to source cheaply (discontinued?) I assume if I got one, it would fit in the U3 location?

What other options do I have? I can't seem to find out what this 101 chip is.
Many thanks

![2022-11-17_083301](https://user-images.githubusercontent.com/16877183/202425424-83b7e9f5-19e5-48df-8cc5-83be483b9d38.jpg)

**Comment by Ralim** [comment]
Hia,
So accelerometers supported by IronOS are:
- BMA223
- LIS2DH12
- MMA8652FC
- MSA301
- SC7A20

At least some of those will fit in the U2 or U3 pads (not certain enough to say).
IronOS should autodetect which one you have once you fit it.

**Reply by mr-bridger** [reply]
Great stuff!, thanks Ralim. 👍😁
